### PR TITLE
issue59 - Create automated performance tests for the extension 

### PR DESCRIPTION
**Pull request by tomekl007**

### CODE REVIEW

File: performance/falloutPerformanceTest.yaml [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+        cloud.instance.type: ms1.small
+        name: dseserver-cassandra-quarkus-perf
+        mark_for_reuse: true
+    configuration_manager:
+      - name: ctool
+        properties:
+          java.version: "1.8_151"
+          product.type: dse
+          product.install.type: tarball
+          product.version: 6.7-dev
```
Comment (adutra) [inline performance/falloutPerformanceTest.yaml:30]: I wonder if OSS Cassandra isn't enough?

File: performance/falloutPerformanceTest.yaml [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+        module: bash
+        properties:
+          target.group: quickstartapp
+          export_output: false
+          timeout: 1 hours
+          script: |
+            cd ${FALLOUT_SCRATCH_DIR}
+            sudo apt update --assume-yes
+            sudo apt-get install lftp --assume-yes
+            sudo apt-get install zlib1g-dev libxmu-dev libxmu-headers freeglut3-dev libxext-dev libxi-dev --assume-yes
```
Comment (adutra) [inline performance/falloutPerformanceTest.yaml:117]: Wow, why do we need all of this?

File: performance/falloutPerformanceTest.yaml [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+        module: bash
+        properties:
+          target.group: http-client
+          export_output: false
+          timeout: 1 hours
+          script: |
+            cd ${FALLOUT_SCRATCH_DIR}
+            sudo apt update --assume-yes
+            sudo apt-get install lftp --assume-yes
+            sudo apt-get install wrk --assume-yes
```
Comment (adutra) [inline performance/falloutPerformanceTest.yaml:165]: Glad to see that it exists in apt!

File: performance/falloutPerformanceTest.yaml [hunk 4]
```diff
@@ -0,0 +1,183 @@
+# branch and version of the cassandra-extension to benchmark
```
Comment (adutra) [inline performance/falloutPerformanceTest.yaml:1]: `fallout.yaml` for the file name is enough imo.

File: performance/falloutPerformanceTest.yaml (hunk 2 above)
Comment (tomekl007) [reply performance/falloutPerformanceTest.yaml:117]: those are needed for native graal compilation

File: performance/falloutPerformanceTest.yaml (hunk 1 above)
Comment (tomekl007) [reply performance/falloutPerformanceTest.yaml:30]: If we would like to test graph or continuous paging in the future maybe it is better to use DSE

**Pull request merged**